### PR TITLE
Good Job

### PR DESCRIPTION
**Pull request by YehorHeiko**
- [DEMO LINK](https://YehorHeiko.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://YehorHeiko.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,8 +6,35 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="header">
+      <div class="container">
+        <img src="./images/imac.jpeg" alt="Apple iMac 27-inch Retina 5K Monoblock"
+        class="container__image">
```
Comment (lerastarynets) [inline src/index.html:17]: attrCheklist

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,3 +1,138 @@
...
+ width: 200px;
+ margin: 15px;
+ text-align: center;
+ border-radius: 20px;
+ padding-top: 16px;
+ box-shadow: 0 14px 28px rgba(0,0,0,0.25);
+
+}
+
+.container__image{
```
Comment (lerastarynets) [inline src/styles/index.scss:24]: pls make use of sass nesting for bem blocks in the who file

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls deploy test reports before rerequesting the review

File: src/index.html [hunk 3]
```diff
@@ -6,8 +6,45 @@
...
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="header" data-qa="card">
+      <div class="container">
+        <div class="container__image">
+          <img src="./images/imac.jpeg"
+          alt="Apple iMac 27-inch Retina 5K Monoblock"
+          class="image"
+          >
```
Comment (anastasiiavorobiova) [inline src/index.html:20]:
```suggestion
          <img src="./images/imac.jpeg"
          alt="Apple iMac 27-inch Retina 5K Monoblock"
          class="image"
          >
```
```suggestion
          <img 
            src="./images/imac.jpeg"
            alt="Apple iMac 27-inch Retina 5K Monoblock"
            class="image"
          >
```

File: src/index.html [hunk 4]
```diff
@@ -6,8 +6,45 @@
...
+        </div>
+        <h1 class="container__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```
Comment (anastasiiavorobiova) [inline src/index.html:22]:
```suggestion
        <h2 class="container__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```
We expect to have a lot of cards on the page

File: src/index.html [hunk 5]
```diff
@@ -6,8 +6,45 @@
...
+        </div>
+        <div class="container__price">
+          <div>Price:</div>
+          <div class="strong">$2,199</div>
+        </div>
+
+        <a
+        class="container__button"
+        data-qa="hover"
+        href="#">
```
Comment (anastasiiavorobiova) [inline src/index.html:43]:
```suggestion
          class="container__button"
          data-qa="hover"
          href="#"
        >
```

File: src/styles/index.scss [hunk 6]
```diff
@@ -1,3 +1,142 @@
...
+}
+
+.header{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  background-color: #FFFF;
+
+
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:27]:
```suggestion
```

File: src/styles/index.scss (hunk 2 above)
Comment (anastasiiavorobiova) [reply src/styles/index.scss:37]: Not fixed

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,3 +1,142 @@
...
+  margin-bottom: 4px;
+
+
+}
+
+.container__product-code{
+  display: flex;
+  justify-content: flex-start;
+  margin-bottom: 16px;
+  font-family: Roboto, sans-serif;
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:65]: Consider applying the font family for the entire page to get it inherited by elements

File: src/styles/index.scss [hunk 8]
```diff
@@ -1,3 +1,142 @@
...
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+
+.container__review-text {
+  display: flex;
+  width: 53px;
+  height: 16px;
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:84]:
```suggestion

```
It's bad practice to hardcode the size of the block

File: src/styles/index.scss [hunk 9]
```diff
@@ -1,3 +1,142 @@
...
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 12px;
+  color: #F3F3F3;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  background-color: #00ACDC;
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:131]: Consider using variables for repeating colors

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
The previous comments were not fixed

**Comment by YehorHeiko** [comment]
Я переписал код по другому? Что именно не исправлено. Тесты все пройдены

**Review by l4st1m0za — APPROVED** [review]
Good job, just change the needed comments

**Pull request closed without merging**